build.gradle.kts

```diff
@@ -82,7 +82,12 @@ dependencies {
     includedModImpl("net.fabricmc.fabric-api:fabric-api-base:0.1.3+12a8474cfa")
     includedModImpl("net.fabricmc.fabric-api:fabric-resource-loader-v0:0.2.9+e5d3217f4e")
     includedModImpl("com.github.Ladysnake:Satin:1.5.0")
-    modImplementation("com.gitlab.CDAGaming:fabritone:fabritone~1.16.x-Fabric-SNAPSHOT")
+    modImplementation("com.gitlab.CDAGaming:fabritone:fabric~1.16.3-SNAPSHOT") {
+        exclude(group = "org.lwjgl.lwjgl")
+        exclude(group= "net.java.jinput")
```

Comment:
formatting :/

Ctrl alt L in intellij

Reply:
rip I forgot that committing outside of Intellij doesn't format. I'm an idiot LOL